types/video.js/tslint.json

```diff
@@ -1 +1,11 @@
-{ "extends": "dtslint/dt.json" }
+{
+	"extends": "dtslint/dt.json",
+	"rules":{
+		"no-padding": false,
```

Comment:
Can you elaborate on why these exceptions are necessary? Generally we don't disable lint rules for new types.

Reply:
For convenience, I decided to include JSDocs comments from video.js docs along with the classes, method,  functions etc... but they are not compatible with the default lint rules. I though this would not be a problem since there are definitions doing the same for popular libs like jQuery.

Reply:
Many of those libs have exceptions because the types predate the linter's presence. If they were overhauled like this, we'd prefer they conformed to the lint rules, too.